invenio_app_ils/ui/src/invenio_app_ils/pages/frontsite/Documents/DocumentsDetails/components/DocumentMetadata/components/DocumentInfo.js

```diff
@@ -14,7 +14,7 @@ export class DocumentInfo extends Component {
         <Table.Row>
           <Table.Cell>Languages</Table.Cell>
           <Table.Cell>
-            {this.metadata.languages.map(lang => lang + ',')}
+            {this.metadata.languages.map(lang => lang + ', ')}
```

Comment:
this will always end with a comma

Reply:
I know I have it temporarily until I find a better generic way to display the list of strings